### ISSUE TITLE
Fix/update node.js usage/integration

### ISSUE THREAD

**Issue by raucao** [remotestorage/remotestorage.js#1191]
The TS port (#1175) was merged to master without caring for node.js usage yet. Should be easy to make it work, as we already have a separate entry point for node.js usage (`index.js`). We can probably use https://www.npmjs.com/package/esm for that.

**Comment by skddc** [comment]
I just tagged `2.0.0-alpha.2`, which I successfully used in a node.js program. However, it is currently necessary to access the constructor via `RemoteStorage.default` when requiring it as a CommonJS module. And I'm wondering if the package mentioned above could help with making that nicer for node users.

Then again, RS modules also have to be loaded with `.default` already, so maybe having it be more consistent is better, and maybe nudging node users towards ES modules is also not a bad idea.

Anyone has opinions on this? Theoretically, this could be closed with just a minor documentation change now (about requiring an AJAX or Fetch lib).
